_maps/map_files/Theseus/TGS_Theseus.dmm

```diff
@@ -36334,8 +36331,8 @@
 /obj/structure/bed/chair{
 	dir = 4
 	},
-/obj/effect/landmark/start/marine/cargotechnician,
-/turf/open/floor/mainship{
+<<<<<<< refs/remotes/upstream/master/obj/effect/landmark/start/marine/cargotechnician,
```

Comment:
Bad merge conflict resolution. Poke me if you need help fixing it.

Reply:
Outdated, even though GitHub hasn't marked it as such.

Reply:
This is outdated, even though GitHub hasn't marked it as such. This issue has been fixed.